### PR TITLE
folder structure (for dashboard)

### PR DESCRIPTION
**Pull request by ogxn**
# Checklist

- [x] This PR has been tested
	- [x] Everything still works
	- [x] No major bugs have been made
		- [ ] I have listed minor bugs that have been added
- [x] I have listed all changes that this PR makes
- [x] This PR needs no further work
- [x] Manual restart and/or configuration required post-merge

- [x] This is a major change
- [ ] This is a minor change
<p/>

- [ ] This is a bug fix
- [x] This is an enhancement or new feature
- [ ] Other: 

Folder structure is ready for dashboard implementation

### CODE REVIEW

**Comment by TheSuperGamer20578** [comment]
Lol started 3 commit messages down and then i saw the next one

File: bot/.gitignore [hunk 1]
```diff
@@ -1,132 +1,132 @@
...
+# mkdocs documentation
+/site
+
+# mypy
+.mypy_cache/
+.dmypy.json
+dmypy.json
+
+# Pyre type checker
+.pyre/
```
Comment (TheSuperGamer20578) [inline bot/.gitignore:132]: Why does it say all lines changed?! Also does git let you do gitignores like this?

File: bot/TheSuperGamer205IsTheWorstProgrammerOnEarth.py [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
+        Send that emoji dont ask me why
+        """
+        await ctx.send("😳")
+
+
+def setup(bot):
+    """
+    Initialize cog
+    """
+    bot.add_cog(TheSuperGamer205IsTheWorstProgrammerOnEarth(bot))
```
Comment (TheSuperGamer20578) [inline bot/TheSuperGamer205IsTheWorstProgrammerOnEarth.py:26]: Git is weird on some files ot says "renamed without changes" and here it says the whole files has changed

File: bot/attack.py [hunk 3]
```diff
@@ -1,119 +1,119 @@
...
+            if len(self.invading[enemy]) > 0:
+                await self.bot.get_channel(731402228820213882).send(f"<@&{borders[enemy]['id']}>", tts=True, delete_after=0)
+        return await self.loop()
+
+
+def setup(bot):
+    """
+    Loads stuff so that it works
+    """
+    bot.add_cog(attack(bot))
```
Comment (TheSuperGamer20578) [inline bot/attack.py:119]: I think these have turned into comments for github

File: bot/autoroles.py [hunk 4]
```diff
@@ -1,62 +1,62 @@
...
+                    await message.author.edit(nick=f"{res.name.replace('_', ' ')} | {res.nation.name.replace('_', ' ')}")
+                else:
+                    await message.author.edit(nick=f"{res.name.replace('_', ' ')}")
+
+
+def setup(bot):
+    """
+    Load stuff so that stuff works
+    """
+    bot.add_cog(autoroles(bot))
```
Comment (TheSuperGamer20578) [inline bot/autoroles.py:62]: Why!

File: bot/core.py [hunk 5]
```diff
@@ -1,207 +1,207 @@
...
+    bot.add_cog(core(setup_bot))
+
+
+if __name__ == '__main__':
+    bot = commands.Bot(command_prefix=("&", "/", ".", "sb!", "s!"))  # , commands.when_mentioned))
+    bot.add_cog(core(bot))
+    for cog in config["general"]["autoload cogs"].split("\n"):
+        if cog != "":
+            bot.load_extension(cog)
+    bot.run(config["api"]["discord"])
```
Comment (TheSuperGamer20578) [inline bot/core.py:207]: Im just going to stop reveiw here and assume its more of the same i dont have time for this

**Review by TheSuperGamer20578 — COMMENTED** [review]
Github can be annoying ill give it a closer look tomorrow

**Comment by TheSuperGamer20578** [comment]
Have you actually tested that this isnt a breaking change? You ticked that you have tested

File: bot/.gitignore (hunk 1 above)
Comment (ogxn) [reply bot/.gitignore:132]: yes you can do gitignores under folders

**Review by TheSuperGamer20578 — CHANGES_REQUESTED** [review]
I don't see a start.sh change without one the bot won't be able to run forever

**Comment by TheSuperGamer20578** [comment]
image
why?

**Comment by ogxn** [comment]
> Have you actually tested that this isnt a breaking change? You ticked that you have tested

everything bot-related has been moved so there is no testing to be done

**Comment by TheSuperGamer20578** [comment]
> > Have you actually tested that this isnt a breaking change? You ticked that you have tested
> 
> everything bot-related has been moved so there is no testing to be done

but with moving there is a possibility that something references something wrongly so ill test it before merge

**Review by TheSuperGamer20578 — COMMENTED** [review]
should work now I just need to run it and make sure it still works and then I can approve

**Review by TheSuperGamer20578 — APPROVED** [review]

**Comment by TheSuperGamer20578** [comment]
I thought that would approve the resolution to my prior review not approve the whole pr

**Review by TheSuperGamer20578 — APPROVED** [review]
This appears to work apart from music however I think that is just my enviroment

**Comment by TheSuperGamer20578** [comment]
I can merge once required check has passed

**Review by TheSuperGamer20578 — APPROVED** [review]
Time to merge!

**Comment by TheSuperGamer20578** [comment]
this is the first pr on this repo to not need me to use admin perms for merge
image

**Comment by ogxn** [comment]
re-merged in #15

**Pull request merged**